sdk/servicebus/service-bus/README.md

```diff
@@ -6,7 +6,6 @@ Use the client library `@azure/service-bus` in your application to
 
 - Send messages to an Azure Service Bus Queue or Topic
 - Receive messages from an Azure Service Bus Queue or Subscription
-- Manage Subscription rules
```

Comment:
Does this need a changelog entry, @ramya-rao-a? 
(Since this is different from track 1)

Reply:
done